### ISSUE TITLE
Harness Slack is not available to me

### ISSUE THREAD

**Issue by Hackwar** [harness/gitness#3368]
I have support questions and wanted to join Slack, as is advertised below all documentation posts. Clicking on the link does not allow me to register/request access to the Harness Community Slack.

**Comment by jimsheldon** [comment]
Are you referring to the "chatroom" links at the bottom of docs on docs.drone.io, or something else?

That link should bring you to https://harnesscommunity.slack.com/signup#/domain-signup, is that where you are having trouble?

**Comment by jimsheldon** [comment]
I think I understand your concern. Currently the community slack requires a Google, Apple or harness.io email account. I'm sure this was done to reduce spam and auto-created accounts.

We are investigating how other slack communities handle signups to see if we can improve on this process. Let us know if you have any suggestions.

**Comment by fanuelsen** [comment]
I also have this problem even with my gmail account I can't join

**Comment by bradrydzewski** [comment]
ability to use a non-Harness email has been fixed.  It was caused due to some weird policy where Slack expires the invitation link after 400 clicks.  We generated a new link and added here https://docs.gitness.com/support

We are working on getting Slack to generate a link that will expire after 4000 clicks, which is better, but unfortunately this problem may crop up every so often unless Slack provides an option for non-expiring links.
